chainerx/_docs/routines.py

```diff
@@ -1149,6 +1149,24 @@ def _docs_math():
     output array to the input array ``x``.
 
 .. seealso:: :data:`numpy.tanh`
+""")
+
+    _docs.set_doc(
+        chainerx.sigmoid,
+        """sigmoid(x)
+Element-wise sigmoid logistic function.
+
+Args:
+    x (~chainerx.ndarray): Input array.
+
+Returns:
+    :class:`~chainerx.ndarray`: Returned array: :math:`f(x) = (1 + \\exp(-x))^{-1}`
```

Comment:
Could you fix the style error? https://travis-ci.org/chainer/chainer/jobs/519521073